src/shared/components/Searchbar/Searchbar.style.tsx

```diff
@@ -39,25 +40,47 @@ export const CancelButton = styled(IconButton)`
   }
 `
 
-export const Container = styled.div`
-  position: relative;
+export const Container = styled.div<{ hasFocus: boolean; hasQuery: boolean }>`
```

Comment:
<img width="518" alt="Screenshot 2021-10-11 at 09 03 09" src="https://user-images.githubusercontent.com/12646744/136746475-321c19a3-6890-4fbd-9855-7a753e372acb.png">

The searchbar has a 2px border now, 1px from `border` and 1px from `box-shadow`. I suggest we use only `box-shadow`